Yafc.UI/ImGui/ImGuiBuilding.cs

```diff
@@ -171,6 +171,7 @@ public partial class ImGui {
         public bool mousePresent { get; private set; }
         public Rect mouseDownRect { get; private set; }
         public Rect mouseOverRect { get; private set; } = Rect.VeryBig;
+        public Rect focusRect { get; private set; }
```

Comment:
Would be good to have a comment that explains the purpose of this variable.

Reply:
totally forgot, sorry